lib/board.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  DIM = 3
+  SIZE = DIM*DIM
+
+  def initialize
+    self.reset!
+  end
+
+  def reset!
+    @cells = %w(\ ) * SIZE
+  #https://stackoverflow.com/questions/1274675/what-does-warray-mean
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    index = input.to_i-1
+    cells[index]
+  end
+
+  def full?
+    @cells.all? do |element|
+      element == "X" || element == "O"
+    end
+  end
+
+  def turn_count
+    @cells.count{|token| token == "X" || token == "O"}
+  end
+
+  def current_player
+    turn_count % 2 == 0 ? "X" : "O"
```

Comment:
Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`

Also, this method is technically game logic and not board logic.  Any particular reason you placed it here?